app/views/widgets/checkboxGroupWithTogglePanels.scala.html

```diff
@@ -0,0 +1,30 @@
+
+@import helpers.CSRSkinnyFieldConstructor._
+
+@(field: play.api.data.Field, options: Seq[(String,String)], args: (Symbol,Any)*)(togglePanels:Html)
+
+@createId(id:String, v:(String,String))=@{id + "_"+v._1.replace(" ", "_").replace("/", "_").replace("'","_")}
```

Comment:
Basic id creation/formatting seems to happen in loads of places. Can consider moving this into a helper if it is necessary? Perhaps include the helpers in the Microservice.scala sbt file in the same way as we do for fset-faststream-frontend. something like
`  lazy val playSettings : Seq[Setting[_]] = Seq(routesImport ++= Seq("binders.CustomBinders._", "models._", "model.Helpers._"))`